src/lamplib/src/genny/tasks/dry_run.py

```diff
@@ -24,6 +24,7 @@ def dry_run_workload(
         "MixedWorkloadsGennyStress.yml",
         "ClusteredCollection.yml",
         "ClusteredCollectionLargeRecordIds.yml",
+        "CreateIndex.yml",
```

Comment:
Let's not do this and see if anything breaks as a way to validate that we actually still need to do this